packages/tools/pluggable-widgets-tools/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mendix/pluggable-widgets-tools",
-  "version": "9.0.3",
+  "version": "9.2.0",
```

Comment:
are we supposed to jump from 9.0.3 to 9.2.0?

Reply:
I think it should even be 9.3.0 by now, we didn't finish the Filtering API for 9.2 :)

Reply:
It will be 9.3 actually 😅 it was supposed to follow mendix releases but sometimes we do not have updates in the piw-tools enough to release